pkg/config/config_template.yaml

```diff
@@ -1026,6 +1026,24 @@ api_key:
   ## The total combined usage all integrations logs files can use
   #
   # integrations_logs_total_usage
+  
+  ## @param streaming - custom object - optional
+  ## This section allows you to configure streaming logs via remote config.
+  # streaming:
+
+    ## @param enable_streamlogs - bool - optional - default: false
+    ## @env DD_LOGS_CONFIG_STREAMING_ENABLE_STREAMLOGS - bool - optional - default: false  
+    ## Enable streaming logs via remote config.
```

Comment:
I probably misunderstand something, but I think this description is inaccurate. If I understand the code here correctly, changing this setting would affect whether flares get a snippet of the logs, but doesn't affect the ability of remote config to also enable that.